packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.tsx

```diff
@@ -55,12 +55,18 @@ const Container = styled.div`
 
 function PageContentWide(props: TPageContentWide) {
   const [leftChild, rightChild] = Children.toArray(props.children);
+  const childrenCount = Children.count(props.children);
 
   useWarning(
-    props.columns !== '1' || !Boolean(rightChild),
+    props.columns === '1' && childrenCount > 1,
     'PageContentWide: This component only renders its first children when using a single column but you provided more that one.'
   );
 
+  useWarning(
```

Comment:
Maybe it would be a good idea to add a test to check this warning the same way we do for the other one.

Reply:
Added here - [c7557ec](https://github.com/commercetools/merchant-center-application-kit/pull/3127/commits/c7557ecb5477db0f99acf7e076887da7249ff3e8)